### PR TITLE
RDM-10523 : Creating index event_case_field_event_id_case_field_id_idx

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-10523

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/ccd-definition-store-api/values.preview.template.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ java:
     #ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1216-es-master
 
 
-    #USER_PROFILE_HOST: http://ccd-user-profile-api-pr-343.service.core-compute-preview.internal
+    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-343.service.core-compute-preview.internal
```
Comment (MSancaktutar) [inline charts/ccd-definition-store-api/values.preview.template.yaml:27]: needs disabling

File: repository/src/main/resources/db/migration/V0002__RDM-10523.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS event_case_field_event_id_case_field_id_idx ON public.event_case_field USING btree (event_id, case_field_id);
```
Comment (MSancaktutar) [inline repository/src/main/resources/db/migration/V0002__RDM-10523.sql:1]: file name convention is now decided. pls check channels for it.

Comment (sumansreenivasan) [reply repository/src/main/resources/db/migration/V0002__RDM-10523.sql]: Renamed the file.

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**